packages/android_alarm_manager_plus/example/integration_test/android_alarm_manager_plus_test.dart

```diff
@@ -2,19 +2,227 @@
+        final validTime = DateTime.utc(1993);
+        const validId = 1;
+
+        // Callback should take a single int param.
+        await expectLater(
+          () => AndroidAlarmManager.oneShotAt(
+            validTime,
+            validId,
+            invalidCallback,
+          ),
+          throwsAssertionError,
+        );
+
+        // ID should be less than 32 bits.
+        await expectLater(
+          () => AndroidAlarmManager.oneShotAt(
+            validTime,
+            2147483648,
+            validCallback,
+          ),
+          throwsAssertionError,
+        );
+      });
+
+      test('sends arguments to the device', () async {
+        final alarm = DateTime.now();
+        const id = 1;
+        const alarmClock = true;
+        const allowWhileIdle = true;
+        const exact = true;
+        const wakeup = true;
+        const rescheduleOnReboot = true;
+
+        final result = await AndroidAlarmManager.oneShotAt(
+          alarm,
+          id,
+          AlarmHelpers.alarmTest,
+          alarmClock: alarmClock,
+          allowWhileIdle: allowWhileIdle,
+          exact: exact,
+          wakeup: wakeup,
+          rescheduleOnReboot: rescheduleOnReboot,
+        );
+
+        expect(result, isTrue);
+      });
+    });
+
+    group('oneShot', () {
+      test('sends arguments to the device', () async {
+        const delay = Duration(milliseconds: 100);
+        const id = 1;
+        const alarmClock = true;
+        const allowWhileIdle = true;
+        const exact = true;
+        const wakeup = true;
+        const rescheduleOnReboot = true;
+
+        final result = await AndroidAlarmManager.oneShot(
+          delay,
+          id,
+          AlarmHelpers.alarmTest,
+          alarmClock: alarmClock,
+          allowWhileIdle: allowWhileIdle,
+          exact: exact,
+          wakeup: wakeup,
+          rescheduleOnReboot: rescheduleOnReboot,
+        );
+
+        expect(result, isTrue);
+      });
+
+      testWidgets('callback is called', (WidgetTester tester) async {
+        await tester.runAsync(() async {
+          WidgetsFlutterBinding.ensureInitialized();
+
+          // Register the UI isolate's SendPort to allow for communication from the
+          // background isolate.
+          port = ReceivePort();
+          IsolateNameServer.registerPortWithName(
+            port.sendPort,
+            isolateName,
+          );
+
+          await tester.pumpWidget(const AlarmManagerExampleApp());
+          tester.tap(find.byKey(const ValueKey('RegisterOneShotAlarm')));
+
+          // 10 seconds is the smallest amount of time we need to wait for
+          // the callback to happen
+          await Future.delayed(const Duration(seconds: 10));
+          await tester.pumpAndSettle();
+
+          expect(
+            find.byWidgetPredicate(
+              (Widget widget) =>
+                  widget is Text &&
+                  widget.data!.startsWith('Alarm fired 1 times'),
```

Comment:
You could try adding `skipOffstage: false,` for checking if it has something to do with the screen size when using the emulator runner (`reactivecircus/android-emulator-runner@v2`) in our GitHub workflow.
It also works locally for me.

Reply:
Yup I can give this a try.

Reply:
Hmm no luck.